common/lib/xmodule/xmodule/tests/test_sequence.py

```diff
@@ -111,6 +111,11 @@ def test_student_view_first_child(self):
     def test_student_view_last_child(self):
         html = self._get_rendered_student_view(self.sequence_3_1, requested_child='last')
         self._assert_view_at_position(html, expected_position=3)
+        html = self._get_rendered_student_view(
+            self.sequence_3_1,
+            extra_context=dict(next_url='NextSequential', prev_url='PrevSequential', efischer_silly_var=True),
+        )
+        self._assert_view_at_position(html, expected_position=1)
```

Comment:
Looking at the code, I believe the issue is that the calls to `_set_up_module_system` and `_set_up_block` should be moved to the `setUp` method from the `setUpClass` method.  The module-system and runtime for the blocks are not getting re-initialized at the beginning of each test.